### ISSUE TITLE
how to load the trained model and use it to predict on a second test set?

### ISSUE THREAD

**Issue by WeiJunZhou23** [HICAI-ZJU/KANO#13]
Hi,
If I have trained a model, how can I reload the saved model and use it on other datasets?
I'm kinda confused about how to use this function `build_model` (let's say I have a trained model: `model_path='KANO/KANO-main/dumped/0729-finetune/activity/run_0/model_2/model.pt'`, `classification`, `'num_tasks': 1`)

Thank you.
